src/unstract/sdk/adapters/ocr/google_document_ai/src/google_document_ai.py

```diff
@@ -110,13 +114,15 @@ def _get_input_file_type_mime(self, input_file_path: str) -> str:
         return file_type_mime
 
     def process(
-        self, input_file_path: str, output_file_path: Optional[str] = None
+        self,
+        input_file_path: str,
+        output_file_path: Optional[str] = None,
+        fs: FileStorage = FileStorage(provider=FileStorageProvider.Local),
     ) -> str:
         try:
             file_type_mime = self._get_input_file_type_mime(input_file_path)
-            if os.path.isfile(input_file_path):
-                with open(input_file_path, "rb") as fop:
-                    file_content_in_bytes: bytes = fop.read()
+            if fs.exists(input_file_path):
```

Comment:
is this `fs.exists` functioning same function of isfile? I think `isfile` returns False for Directory, hope the same behaviour in `exists`

Reply:
Currently, exists will only indicate if the object path exists.